packages/odata-service-writer/src/index.ts

```diff
@@ -148,7 +148,8 @@ async function generate(basePath: string, service: OdataService, fs?: Editor): P
         updatePackageJson(paths.packageJson, fs, !!service.metadata);
     }
 
-    if (ui5LocalConfigPath && ui5LocalConfig) {
+    if (!serviceIsCds(service) && ui5LocalConfigPath && ui5LocalConfig) {
```

Comment:
```suggestion
    if (service?.serviceType === Servicetype.CDS && ui5LocalConfigPath && ui5LocalConfig) {
```

No need for the indirection?